gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-ksp = "1.8.20-1.0.11"
+ksp = "1.9.0-1.0.12"
 kotlinJupyter = "0.11.0-358"
 ktlint = "3.4.5"
-kotlin = "1.8.20"
+kotlin = "1.9.0"
```

Comment:
Did you test, does it compile well?

Reply:
Actually, no :) Something's broken in gradle-plugin module

Reply:
last commit resolved the issue